### ISSUE TITLE
twisted.names doesn't set authority bit

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11468]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#106|
|Type|defect|
|Created|2003-07-19 13:37:29Z|

Attachments:

* [names-authority.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/722/7224f997fc148baa0b7f81c1eda6fcc3fd003db0/927834362c7cf3301119f5655a20dd4a0c1d5860.patch) (4626 bytes) - added by radix on 2003-07-20 02:52:55Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__106 106
type__defect defect
reporter__radix radix
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__ 
time__1058621849000000 1058621849000000
changetime__1465876435831919 1465876435831919
version__ 
owner__ 
cc__radix cc__exarkun
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
Resolvers need to be able to specify whether they're
authoritative to the lookup they're responding to. They
can't, currently.

</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
I've implemented this and I'm testing it on divmod.net.

</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
Ok, it seemed to work. I've committed it.

</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|GitHub commented|
|-|-|

In changeset 1a0e1d1adcaa3913704b9868d80d1a15bb41dfcf
```
#!CommitTicketReference repository="" revision="1a0e1d1adcaa3913704b9868d80d1a15bb41dfcf"
Merge pull request #106 from rodrigc/8390-rodrigc-assert

Author: rodrigc

Reviewer: glyph

Fixes: #8390

Clean up a few assertion methods in twisted.runner's test suite:

- `assertIsNone` rather than our undocumented `assertIdentical`
- more expressive `assertIn` rather than `assertTrue`
```
